src/DocumentFormat.OpenXml/Packaging/OpenXmlSupportedRelationshipExtensions.cs

```diff
@@ -0,0 +1,312 @@
+    /// </summary>
+    /// <param name="parent">The parent part requesting to add.</param>
+    /// <param name="contentType">The content type information for the added extensible part.</param>
+    /// <param name="id">The relationship id. Optional, default to null.</param>
+    /// <return>The newly added part</return>
+    /// <see cref="EmbeddedControlPersistencePart"/>
+    public static EmbeddedControlPersistencePart AddEmbeddedControlPersistencePart<T>(this T parent, string contentType, string? id = null)
+        where T : OpenXmlPartContainer, ISupportedRelationship<EmbeddedControlPersistencePart>
+        => DoInitPart(parent, new EmbeddedControlPersistencePart(), contentType, id);
+
+    /// <summary>
+    /// Adds a EmbeddedObjectPart as a relationship to the parent part
+    /// </summary>
+    /// <param name="parent">The parent part requesting to add.</param>
+    /// <param name="partType">The part type information for the added extensible part.</param>
+    /// <param name="id">The relationship id. Optional, default to null.</param>
+    /// <return>The newly added part</return>
+    /// <see cref="EmbeddedObjectPart"/>
+    public static EmbeddedObjectPart AddEmbeddedObjectPart<T>(this T parent, PartTypeInfo partType, string? id = null)
+        where T : OpenXmlPartContainer, ISupportedRelationship<EmbeddedObjectPart>
+        => InitPart(parent, new EmbeddedObjectPart(), partType, id);
+
+    /// <summary>
+    /// Adds a EmbeddedObjectPart as a relationship to the parent part
+    /// </summary>
+    /// <param name="parent">The parent part requesting to add.</param>
+    /// <param name="contentType">The content type information for the added extensible part.</param>
+    /// <param name="id">The relationship id. Optional, default to null.</param>
+    /// <return>The newly added part</return>
+    /// <see cref="EmbeddedObjectPart"/>
+    public static EmbeddedObjectPart AddEmbeddedObjectPart<T>(this T parent, string contentType, string? id = null)
+        where T : OpenXmlPartContainer, ISupportedRelationship<EmbeddedObjectPart>
+        => DoInitPart(parent, new EmbeddedObjectPart(), contentType, id);
+
+    /// <summary>
+    /// Adds a EmbeddedPackagePart as a relationship to the parent part
+    /// </summary>
+    /// <param name="parent">The parent part requesting to add.</param>
+    /// <param name="partType">The part type information for the added extensible part.</param>
+    /// <param name="id">The relationship id. Optional, default to null.</param>
+    /// <return>The newly added part</return>
+    /// <see cref="EmbeddedPackagePart"/>
+    public static EmbeddedPackagePart AddEmbeddedPackagePart<T>(this T parent, PartTypeInfo partType, string? id = null)
+        where T : OpenXmlPartContainer, ISupportedRelationship<EmbeddedPackagePart>
+        => InitPart(parent, new EmbeddedPackagePart(), partType, id);
+
+    /// <summary>
+    /// Adds a EmbeddedPackagePart as a relationship to the parent part
```

Comment:
```suggestion
    /// Adds a <see cref="EmbeddedPackagePart" /> as a relationship to the parent part
```

and elsewhere in the doc